react/react1/week1/todo-app/src/components/Todos.js

```diff
@@ -0,0 +1,30 @@
+const todoItem = [
```

Comment:
The naming isn't quite proper, this is an array, so it contains items. Maybe you can call it todoItems instead.
```suggestion
const todoItems = [
```